app/app.js

```diff
@@ -71,7 +71,11 @@ app.use(addRequestId);
 // app.options('*', cors());
 app.use(cors());
 app.use(coreLib.middleware.logger(null, logger));
-app.use(bodyParser.json());
+app.use(bodyParser.json({
+  verify: (req, res, buf) => {
+    req.rawBody = buf.toString();
```

Comment:
@jkeam Would you mind sharing why we need this piece of code?

Reply:
Yep!  We are using it here:

```
const token = SecurityHelper.calculateHmac(req.rawBody, 'sha256=');
```

Essentially we need access to the raw body in order to generate the sha256 HMAC of the raw payload.  This will allow us to ensure that the webhook was generated from the discourse app that we expected.

Reply:
Okay. Hope, it won't affect other endpoints working.

Reply:
It shouldn't.  `rawBody` doesn't exist until this line.  And it isn't used by anything else.